db/migrate/20171212101253_add_submit_external_to_message_thread.rb

```diff
@@ -0,0 +1,9 @@
+class AddSubmitExternalToMessageThread < ActiveRecord::Migration
+  def change
+    add_column :message_threads, :external_service_id, :integer
+    create_table :external_services do |t|
+      t.string :name, null: false
+      t.string :short_name, null: false
+    end
+  end
+end
```

Comment:
In pulls it helps if you also run `rake db:migrate` and commit the schema changes.